mne_bids/utils.py

```diff
@@ -347,3 +354,198 @@ def _read_events(events_data, raw):
     else:
         events = find_events(raw, min_duration=0.001)
     return events
+
+
+def _get_brainvision_paths(vhdr_path):
+    """Get the .eeg and .vmrk file paths from a BrainVision header file.
+
+    Parameters
+    ----------
+    vhdr_path : str
+        path to the header file
+
+    Returns
+    -------
+    paths : tuple
+        paths to the .eeg file at index 0 and the .vmrk file
+        at index 1 of the returned tuple
+
+    """
+    fname, ext = _parse_ext(vhdr_path)
+    if ext != '.vhdr':
+        raise ValueError('Expecting file ending in ".vhdr",'
+                         ' but got {}'.format(ext))
+
+    # Header file seems fine, read it
+    with open(vhdr_path, 'r') as f:
+        lines = f.readlines()
+
+    # Try to find data file .eeg
+    eeg_file_match = re.search(r'DataFile=(.*\.eeg)', ' '.join(lines))
+    if not eeg_file_match:
+        raise ValueError('Could not find a .eeg file link in'
+                         ' {}'.format(vhdr_path))
+    else:
+        eeg_file = eeg_file_match.groups()[0]
+
+    # Try to find marker file .vmrk
+    vmrk_file_match = re.search(r'MarkerFile=(.*\.vmrk)', ' '.join(lines))
+    if not vmrk_file_match:
+        raise ValueError('Could not find a .vmrk file link in'
+                         ' {}'.format(vhdr_path))
+    else:
+        vmrk_file = vmrk_file_match.groups()[0]
+
+    # Make sure we are dealing with file names as is customary, not paths
+    # Paths are problematic when copying the files to another system. Instead,
+    # always use the file name and keep the file triplet in the same directory
+    assert os.sep not in eeg_file
```

Comment:
can these paths be in different folder hierarchies at recording? if so, maybe we should have a better error handling here

Reply:
For both BrainVision and EEGLAB we can have arbitrary pointers to files. When these files are written by e.g., BrainVision Recorder of saved with EEGLAB the file pointers will always be simply the file name and we assume the file that is being pointed to to be in the same directory.

*However*, user might manually change the pointers and include a path instead of only the file name. That is something I have not yet encountered ... but it is technically possible.